### ISSUE TITLE
[named-export] False positives when exported name does not strictly follow PascalCase.

### ISSUE THREAD

**Issue by epaew** [epaew/eslint-plugin-filenames-simple#156]
## Describe the bug
ESLint detects the warning `The export name must match the filename. You need to rename to EstreeParser or estreeParser  filenames-simple/named-export` with the following code.

- src/utils/eslint-parser/intex.ts
    ```typescript
      export class ESTreeParser {
      ...
    ```

## Expected behavior
No warning is detected.

### Versions
* Node.js: 12.16.3
* ESLint: 7.11.0
* plugin: 0.4.0

### Your .eslintrc.{js,json,yaml}
https://github.com/epaew/eslint-plugin-filenames-simple/blob/d4669182fd95475091ef14df69e0f5bd928b10d2/.eslintrc.yaml

**Comment by epaew** [comment]
You can lint variable names using the following rules ([eslint/camelcase](https://eslint.org/docs/rules/camelcase), [@typescript-eslint/naming-convention](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md), etc.),
and lint filenames using the rule [filenames-simple/naming-convention](https://github.com/epaew/eslint-plugin-filenames-simple/blob/master/docs/rules/naming-convention.md).

So I'd like to change my comparison strategy to simply compare lowercased names without `-`,` _`.
